packs/feats/conceal-spell.json

```diff
@@ -49,7 +49,7 @@
                 "property": "description",
                 "value": [
                     {
-                        "text": "PF2E.SpecificRule.Spellshape.ConcealSpell"
+                        "text": "PF2E.SpecificRule.Spellshape.ConcealSpell.Effect"
```

Comment:
This might need to be migrated?

Reply:
Hmm - how much of a hassle would this be?

Reply:
Any alternatives on how to do this without having to go through a migration?

Reply:
Not too bad - you can look at some of the other migrations to see examples.

Reply:
Can you point me where to look at?
Kinda new to the concept of migrations.

Thank you again for walking me through this

Reply:
You'll want a migration that `extends MigrationBase`. Take a look at Migration924JiuHuanDoa. You'll need to test it's working with `npm run migrate`.

Reply:
Also, `recursiveReplaceString() might be of some use.